chain/src/view.rs

```diff
@@ -71,7 +71,7 @@ pub trait View: StateExt {
     /// Gets the current block timestamp from the JMT
     async fn get_block_timestamp(&self) -> Result<Time> {
         let timestamp_string: String = self
-            .get_proto(b"block_timestamp".into())
+            .get_proto(state_key::block_height())
```

Comment:
this should be block_timestamp instead of block_height

Reply:
oh blocker noticed now...sorry!